File: atcoder/ABC338A_capitalized.cpp
// Vicfred
// https://atcoder.jp/contests/abc338/tasks/abc338_a
// implementation
#include <iostream>

using namespace std;

int main() {
  string S;
  cin >> S;
  bool valid = true;
  if((S[0] - 'A' + 1) > 26) {
    valid = false;
  }
  for(int i = 1; i < S.size(); ++i) {
    if((S[i] - 'A' + 1) <= 26) {
      valid = false;
      break;
    }
  }
  if(valid) {
    cout << "Yes" << endl;
  } else {
    cout << "No" << endl;
  }
  return 0;
}

File: atcoder/abc338a_capitalized.hs
-- vicfred
-- https://atcoder.jp/contests/abc338/tasks/abc338_a
-- implementation
import Data.Bool (bool)
import Data.Char (isLower)

main :: IO ()
main = getLine >>= putStrLn . bool "No" "Yes" . f

f :: String -> Bool
f (x:xs) = if (x `elem` ['A'..'Z']) && (all isLower xs) then True else False
